ui/app/models/auth-method.js

```diff
@@ -85,9 +87,9 @@ const ModelExport = AuthMethodModel.extend({
       ];
     }
     return expandAttributeMeta(this, tuneAttrs);
-  }),
+  }
 
-  formFields: computed(function () {
+  get formFields() {
```

Comment:
I remember refactoring formFields and formFieldGroups to getters causing issues with the `secret-engine` model and some additional changes like 
```js
 get fieldGroups() {
    return this._expandGroups(this.formFieldGroups);
  }
```

It seems like we have have issues specifically with `tuneAttrs` since those rely on the auth method type. I can't remember exactly how the issue surfaced, I think it was in the edit form when changing between methods (selecting enable 'token' then clicking back to enable a different method type?)

Reply:
Quickly tested and from what I can tell it looks like the attrs update correctly since it's a separate request made after select + enable

Reply:
Yeah, since it's based on an attribute that isn't changing within the form (only on mount) I think we're safe here. But good check!